Formula/fop.rb

```diff
@@ -20,8 +14,13 @@ class Fop < Formula
   end
 
   def install
+    rm_rf Dir["fop/*.bat"] # Remove Windows files.
     libexec.install Dir["*"]
-    bin.write_exec_script libexec/"fop"
+
+    executable = libexec/"fop/fop"
+    executable.chmod 0555
```

Comment:
Presume upstream screwed up the permissions on the shell script in this release, given it worked fine before.